### PR TITLE
Add a non verbose (quiet) mode for rosnode info

### PR DESCRIPTION
**Pull request by spaghetti-**
Addresses https://github.com/ros/ros_comm/issues/804

Default behavior not changed to avoid regression.

### CODE REVIEW

**Comment by spaghetti-** [comment]
@dirk-thomas This file contains a lot of trailing whitespace which my editor removed by default, but I figured I'd open another pull request for that it makes reading this particular diff pretty annoying. Do let me know if you're interested.

https://github.com/spaghetti-/ros_comm/commit/0d51e75032a3e6ea824018d928119f1a759ac36b

File: tools/rosnode/src/rosnode/__init__.py [hunk 1]
```diff
@@ -554,7 +554,7 @@ def get_node_connection_info_description(node_api, master):
         raise ROSNodeIOException("Communication with node[%s] failed!"%(node_api))
     return buff
 
-def rosnode_info(node_name):
+def rosnode_info(node_name, non_verbose):
```
Comment (dirk-thomas) [inline tools/rosnode/src/rosnode/__init__.py]: Please avoid inverted logic. Either call the argument `quiet` or `verbose`.

Adding a positional argument breaks existing code. Please add the new argument as an optional keyword argument. And the default value should not change the current behavior.

File: tools/rosnode/src/rosnode/__init__.py [hunk 2]
```diff
@@ -613,12 +613,17 @@ def _rosnode_cmd_info(argv):
     Implements rosnode 'info' command.
     """
     args = argv[2:]
-    parser = OptionParser(usage="usage: %prog info node1 [node2...]", prog=NAME)
+    parser = OptionParser(usage="usage: %prog info [options] node1 [node2...]",
+                          prog=NAME)
+    parser.add_option("-q", "--quiet",
+                      dest="non_verbose", default=False,
```
Comment (dirk-thomas) [inline tools/rosnode/src/rosnode/__init__.py]: Please avoid `dest`. The option name should already be clear enough to map to a variable. Also the `default` argument is not necessary for a boolean action.

File: tools/rosnode/src/rosnode/__init__.py [hunk 3]
```diff
@@ -613,12 +613,17 @@ def _rosnode_cmd_info(argv):
...
+                      action="store_true",
+                      help="non-verbose mode")
```
Comment (dirk-thomas) [inline tools/rosnode/src/rosnode/__init__.py]: Please describe the difference of the output in the help text.

**Comment by dirk-thomas** [comment]
Thanks for avoiding any whitespace changes. The are usually not being considered for merge anyway since it makes backporting changes between branches unnecessary cumbersome.

**Comment by spaghetti-** [comment]
thanks for the comments - fixed.

**Comment by dirk-thomas** [comment]
Thanks for the patch and update.

**Pull request merged**